libbeat/tests/system/config/mockbeat.yml.j2

```diff
@@ -77,6 +77,7 @@ setup.template:
   name: {{ es_template_name }}
   pattern: {{ es_template_pattern }}
   overwrite: {{ template_overwrite|default("false") }}
+  type: {{ template_type|default("index") }}
```

Comment:
Wondering if we should add a test for a legacy template to make sure it keeps working?